.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules/*
 dist/*
 build/*
 .env
+.gitignore
```

Comment:
porque o `.gitignore` não deve ser commitado? e se for esse o caso, ele continua aqui